lisa/tools/lsblk.py

```diff
@@ -155,57 +151,37 @@ def get_disks(self, force_run: bool = False) -> List[DiskInfo]:
         disks: List[DiskInfo] = []
 
         # parse output of lsblk
+        # -b print SIZE in bytes rather than in human readable format
+        # -J output in JSON format
+        # -o list of columns to output
         output = self.run(
-            "-b -P -o NAME,SIZE,TYPE,MOUNTPOINT,FSTYPE", sudo=True, force_run=force_run
+            "-b -J -o NAME,SIZE,TYPE,MOUNTPOINTS,FSTYPE", sudo=True, force_run=force_run
```

Comment:
MOUNTPOINTS don't work on linux. It is `MOUNTPOINT`

Reply:
I tested it with Ubuntu 22.04 WSL. Which distro doesn't work? The MOUNTPOINT returns just one mount point. In WSL, it doesn't return the root path `/` by default.

Reply:
I tested it on marketplace image `Canonical 0001-com-ubuntu-server-focal 20_04-lts latest`

Reply:
ok, I will change it back to MOUNTPOINT for compatibility. It should also work.